gordo_components/workflow/workflow_generator/resources/argo-workflow.yml.template

```diff
@@ -1326,6 +1337,8 @@ spec:
           template: model-builder
           arguments:
             parameters:
+              - name: project-name
+                value: "{{ project_name }}"
```

Comment:
I think you can just remove this (project-name) from the "input" stuff, and just inject it directly in the environment variable section?

Reply:
The same could go for all of the input parameters then. Is there a reason it was done for the other input params?

Reply:
Just for the record: @ryanjdillon and I spoke about this and clarified